app/assets/src/components/PipelineSampleReport.jsx

```diff
@@ -833,13 +894,25 @@ class PipelineSampleReport extends React.Component {
     if (tax_info.tax_id > 0) {
       if (report_details.taxon_fasta_flag) {
         taxonNameDisplay = (
-          <span className="taxon-sidebar-link" onClick={onTaxonClickHandler}>
+          <span
+            className="taxon-sidebar-link"
+            onClick={withAnalytics(
+              onTaxonClickHandler,
+              "PipelineSampleReport_taxon-sidebar-link_clicked"
+            )}
+          >
             <a>{taxonNameDisplay}</a>
           </span>
         );
       } else {
         taxonNameDisplay = (
-          <span className="taxon-sidebar-link" onClick={onTaxonClickHandler}>
+          <span
+            className="taxon-sidebar-link"
+            onClick={withAnalytics(
+              onTaxonClickHandler,
+              "PipelineSampleReport_taxon-sidebar-link_clicked"
+            )}
+          >
             {taxonNameDisplay}
           </span>
         );
```

Comment:
you're probably saving this for later but I've always wondered if people are hovering over the PathogenPreview dots below

Reply:
`PathogenPreview` and `ReportInsightIcon` are tricker to instrument because they don't have event hooks, so I think we should do them later.